WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
Oh, I have sources all right.

> What shall we say then? Are we to continue in sin so that grace may increase? May it never be! How shall we who died to sin still live in it?
>-**Romans 6:1-2**

This was one of the first references to arguments that they could sin consistently to increase and receive grace.

>It is actually reported that there is immorality among you, and immorality of such a kind as does not exist even among the Gentiles, that someone has his father's wife. You have become arrogant and have not mourned instead, so that the one who had done this deed would be removed from your midst. For I, on my part, though absent in body but present in spirit, have already judged him who has so committed this, as though I were present.
>In the name of our Lord Jesus, when you are assembled, and I with you in spirit, with the power of our Lord Jesus, I have decided to deliver such a one to Satan for the destruction of his flesh, so that his spirit may be saved in the day of the Lord Jesus. Your boasting is not good Do you not know that a little leaven leavens the whole lump of dough? Clean out the old leaven so that you may be a new lump, just as you are in fact unleavened. For Christ our Passover also has been sacrificed. Therefore let us celebrate the feast, not with old leaven, nor with the leaven of malice and wickedness, but with the unleavened bread of sincerity and truth.
>-1 Corinthians 5:1-8

This was directly written to the Corinthians, who had Antinomian heretics among them, who decided that because of endless forgiveness, they could go into carnal sin and ask for forgiveness afterwards.